.github/workflows/main.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Setup DotNet on MacOS
         if: runner.environment == 'github-hosted' && runner.os == 'macos'
         run: |
-          dotnet workload install android macos ios
+          dotnet workload install macos ios
```

Comment:
Android builds started failing on macOS again so disabled them.

They only seem to be problematic for macOS builds, Linux and Windows are building and testing Android so not that important.

Reply:
So disabling the .NET SSL check didn't work?

Reply:
Nop :/